flow/flat_buffers.h

```diff
@@ -250,6 +251,29 @@ struct vector_like_traits<std::set<Key, Compare, Allocator>> : std::true_type {
 		return v.begin();
 	}
 };
+template <class Key, class Compare, class Allocator>
```

Comment:
It looks like the template arguments for std::unordered_set are Key, Hash, KeyEqual, and Allocator according to https://en.cppreference.com/w/cpp/container/unordered_set. 
Also would you mind adding a unit test to flow/flat_buffers.cpp?

Reply:
Good catch! Missed that part, given it is a hash based container, following should be used. 

Plus, will add a unit test for it. 
Thanks

Reply:
Test case added.
Thanks